packages/css-library/src/tokens/color.scss

```diff
@@ -9,8 +9,8 @@ $tokens-color-basic: (
   "link-default": $color-link-default,
   "warning-message": $color-warning-message,
   "primary": $color-primary,
+  "primary-dark": $color-primary-dark,
   "primary-darker": $color-primary-darker,
-  "primary-darkest": $color-primary-darkest,
```

Comment:
If this token is being removed/replaced, we might need to check if there are existing usages of it and update those. 

![Screenshot 2023-12-05 at 11 01 59 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/75382257-074d-4bbe-be89-c32104a324f5)

Reply:
oooh yeah good catch. Probably also need to shift all the usages of `--color-primary-darker` up to `--color-primary-dark` before doing the one above.

Reply:
@jamigibbs - this should be fixed now. Because of Matt's comment [here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2228#issuecomment-1836918069), prior and in addition to the changes you mentioned above, I shifted all of the existing `--color-primary-darker' usages (except back to top since it was already correct) up to `--color-primary-dark` since that now contains the old `--color-primary-darker` value, and then after that replaced all `--color-primary-darkest` usages with `--color-primary-darker`.

Reply:
Confirmed hover state only shifts one level down like it does currently:

![image](https://github.com/department-of-veterans-affairs/component-library/assets/15097156/0a50d74a-ba98-4e44-ab3d-e5004338b97b)